cmd/kaniko-docker/main.go

```diff
@@ -94,6 +94,12 @@ func main() {
 			Usage:  "docker password",
 			EnvVar: "PLUGIN_PASSWORD",
 		},
+		cli.StringFlag {
+			Name: "skip_tls_verify",
```

Comment:
use kebab case for flag names (skip-tls-verify)

Reply:
done